libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -99,10 +99,6 @@ extern "C++" {
 #    define _LIBCUDACXX_HAS_STL_LITERALS
 #  endif
 
-#  if _CCCL_COMPILER(GCC) && __cplusplus < 201103L
-#    error "libc++ does not support using GCC with C++03. Please enable C++11"
-#  endif
-
```

Comment:
I believe we should actually change this to
```suggestion
#  if __cplusplus < 201103L
#    error "libcu++ does not support C++03. Please enable C++11"
#  endif
```

Reply:
This check is already implemented several lines before, That's why I want to remove this one completely.